.github/workflows/ssi-authority-schema-registry-migrations.yml

```diff
@@ -0,0 +1,92 @@
+###############################################################
```

Comment:
I suggest to shorten the file, remove the SSI Authority & Schema Registry part.

Reply:
adjusted